File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/customserializers/MyDto.java
package io.micronaut.configuration.openapi.docs.customserializers;

import jakarta.xml.bind.JAXBElement;

// tag::clazz[]
class MyDto {

    public JAXBElement<? extends XmlElement> xmlElement;
    public JAXBElement<? extends XmlElement2> xmlElement2;
    public JAXBElement<? extends XmlElement3> xmlElement3;
}
// end::clazz[]

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/customserializers/MyDto.kt
package io.micronaut.configuration.openapi.docs.customserializers

import jakarta.xml.bind.JAXBElement

// tag::clazz[]
class MyDto {

    var xmlElement: JAXBElement<out XmlElement>? = null
    var xmlElement2: JAXBElement<out XmlElement2>? = null
    var xmlElement3: JAXBElement<out XmlElement3>? = null
}
// end::clazz[]
